server/api.js

```diff
@@ -64,7 +64,7 @@ router.post("/bookings", async function(req, res) {
 
 		if (userResult.rows.length === 0) {
 			return res.status(400).send({
-				message: `User with name ${userName} does not exist`,
+				message: `User with the name ${userName} does not exist`,
```

Comment:
Maybe just "${userName} does not exist!"

Reply:
I was thinking the same!